exporters/elasticsearch-exporter/src/main/resources/zeebe-record-decision-evaluation-template.json

```diff
@@ -0,0 +1,117 @@
+{
+  "index_patterns": [
+    "zeebe-record_decision_evaluation_*"
+  ],
+  "composed_of": [
+    "zeebe-record"
+  ],
+  "priority": 20,
+  "version": 1,
+  "template": {
+    "settings": {
+      "number_of_shards": 1,
+      "number_of_replicas": 0,
+      "index.queries.cache.enabled": false
+    },
+    "aliases": {
+      "zeebe-record-decision-evaluation": {}
+    },
+    "mappings": {
+      "properties": {
+        "value": {
+          "dynamic": "strict",
+          "properties": {
+            "decisionId": {
+              "type": "keyword"
+            },
+            "decisionName": {
+              "type": "keyword"
+            },
+            "decisionVersion": {
+              "type": "long"
+            },
+            "decisionKey": {
+              "type": "long"
+            },
+            "decisionRequirementsId": {
+              "type": "keyword"
+            },
+            "decisionRequirementsKey": {
+              "type": "long"
+            },
+            "decisionOutput": {
+              "enabled": false
+            },
```

Comment:
❌ It's unclear to me why this is disabled. I'd expect that the reason for this would be explained in the commit message.

Reply:
The property `enable: false` disable the indexing of the property. The value itself is exported.
It follows the existing way. For example, of the [job record](https://github.com/camunda-cloud/zeebe/blob/main/exporters/elasticsearch-exporter/src/main/resources/zeebe-record-job-template.json#L61-L63) or [message record](https://github.com/camunda-cloud/zeebe/blob/main/exporters/elasticsearch-exporter/src/main/resources/zeebe-record-message-template.json#L34-L36).